roo.gemspec

```
@@ -23,6 +23,7 @@ Gem::Specification.new do |spec|
     spec.required_ruby_version  = ">= 2.7.0"
   end
 
+  spec.add_dependency 'csv', '~> 3'
```

@andyundso It seems it needs to add dependency of `base64`.

The base64 gem warn same message, warning message details is following:
```
/path/roo/lib/roo.rb:9: warning: base64 was loaded from the standard library, but will no longer be part of the default gems starting from Ruby 3.4.0.
You can add base64 to your Gemfile or gemspec to silence this warning.
```


```suggestion
  spec.add_dependency 'base64' if RUBY_VERSION >= '3.3'
  spec.add_dependency 'csv' if RUBY_VERSION >= '3.3'
```